camdecmps/views/vw_em_eval_and_submit.sql

```diff
@@ -16,37 +16,40 @@ SELECT
         esa.userid,
         COALESCE(esa.update_date, esa.add_date) AS update_date,
         esa.sub_availability_cd AS window_status,
-        prd.period_abbreviation
-    FROM (
-        SELECT
-            ems.mon_plan_id,
-            ems.rpt_period_id,
-            (
-                SELECT
-                    esa.em_sub_access_id
-                FROM (
+        rpt.period_abbreviation
+    FROM
+        camd.plant p
+        JOIN camdecmps.monitor_plan mp ON mp.fac_id = p.fac_id
+        JOIN camdecmps.emission_evaluation ee ON ee.mon_plan_id = mp.mon_plan_id
```

Comment:
This join to EMISSION_EVALUATION is not needed since the "lst" subquery includes the same information.  Additionally the REPORTING_PERIOD will need to join to the subquery results.